app/lib/efile/md/md502_su_calculator.rb

```diff
@@ -3,28 +3,59 @@ module Md
     class Md502SuCalculator < ::Efile::TaxCalculator
       attr_reader :lines, :value_access_tracker
 
+      CODE_LETTERS = ["AB", "U"]
```

Comment:
since we will be adding "v" shortly i tried to loop through the code as much as i could to make things easier to test.